docs/UserGuide.md

```diff
@@ -71,6 +75,25 @@ Shows a message explaining how to access the help page.
 
 Format: `help`
 
+### Listing all Client : `list`
+
+Show a list of all clients contacts in Clientele+.
+
+Format: `list`
+
+### Clearing All Entries : `clear`
+
+Clears all entries from Clientele+.
+
+Format: `clear`
+
+### Exiting the Program : `exit`
+
+Exits the program.
```

Comment:
By any chance, should we include a tip saying that you can also just use the "x" at the top left corner, and the fact that it will auto save for you?

Reply:
Yes, I’ll add that tip now. Thanks for the suggestion!